src/java/org/apache/cassandra/tools/nodetool/stats/CompactionHistoryHolder.java

```diff
@@ -54,8 +54,9 @@ private static class CompactionHistoryRow implements Comparable<CompactionHistor
         private final long bytesIn;
         private final long bytesOut;
         private final String rowMerged;
+        private final String compactionType;
 
-        CompactionHistoryRow(String id, String ksName, String cfName, long compactedAt, long bytesIn, long bytesOut, String rowMerged)
+        CompactionHistoryRow(String id, String ksName, String cfName, long compactedAt, long bytesIn, long bytesOut, String rowMerged, String compactionType)
```

Comment:
@Maxwell-Guo you have not changed name of this property to `compactionProperties`.

Reply:
ok